### ISSUE TITLE
TAB Composition

### ISSUE THREAD

**Issue by pwensing** [pwensing/RAS-Governance#39]
Needs SAC chair on TAB

**Comment by pwensing** [comment]
This Board shall consist of the Chair (Vice President for Technical Activities), the President, all Associate Vice Presidents Technical Activities, the Editors-in-Chief of the Transactions, Letters and Magazine, the Vice President for Conference Activities, the Chairs of the Technical Committees, the Vice President for Media Services, **the Chair of the Student Activities Committee**, plus between one and five additional members appointed by the Vice President for Technical Activities with the approval of the President. At least one member of the board must be an elected member of the AdCom. This Board shall be responsible for coordinating all the technical activities of the Society.

**Comment by pwensing** [comment]
Kyujin requests: amended to include Cluster chairs

**Comment by pwensing** [comment]
The current language is:

> This Board shall consist of the Chair (Vice President for Technical Activities), the President, all Associate Vice Presidents for Technical Activities, the Editors-in-Chief of the Transactions, Letters and Magazine, the Vice President for Conference Activities, the Chairs of the Technical Committees, the Vice President for Media Services, **plus between one and eight additional members** appointed by the Vice President for Technical Activities with the approval of the President. At least one appointed member of the board must be an elected member of the AdCom. This Board shall be responsible for coordinating all the technical activities of the Society.

- [ ] @JenJenChung to circle up with KyuJin regarding whether or not the up to 8 additional seats are adequate to accommodate cluster chairs on the board  -- otherwise, we need to form a more substantial Bylaw update to formalize the Cluster Chairs, how they are appointed, their responsibilities, etc.

**Comment by JenJenChung** [comment]
Contacted Kyujin on 13/8 but no response yet. Should we follow up?

**Comment by pwensing** [comment]
Yeah -- I think so. Will you be at the ExCom mtg today? Perhaps we can follow up there.
